src/docs/operator-guides/migrating-from-outside-sharetribe/index.md

```diff
@@ -73,32 +146,65 @@ types:
 
 This means that the Intermediary supports importing:
 
-- Users
-- Listings
-- Profile images
-- Listing images
-- Stripe accounts - You need to use the same Stripe keys in Flex as in
-  your current service
-- Reviews
-
-Supported references between the different types are:
+- [Users](#user)
+- [Listings](#listing)
+- [Profile images](#image)
+- [Listing images](#image)
+- [Stripe accounts](#stripe-account) - You need to use the same Stripe
+  keys in Flex as in your current service
+- [Reviews](#review)
+
+The migration process has some built-in validation for the types within
+the Intermediary file. Optional keys must have non-empty values, and if
+an optional key does not have a value, the key must be omitted for that
+resource. Empty strings are validated as empty values.
+
+#### Listing
+
+Supported fields to migrate:
+[API reference listing resource](https://www.sharetribe.com/api-reference/marketplace.html#listing-resource-format)
+
+- Required attributes
```

Comment:
In the attributes list we could state the type of the data too. For example that `:im.listing/createdAt` is an [#inst](https://github.com/edn-format/edn#inst-rfc-3339-format) tagged element, title is a string, and so on.